docs/docs/overview/quickstart/get-started-with-dotnet.md

```diff
@@ -0,0 +1,359 @@
+
+- The value of `EventName` should be the notification template’s trigger ID/slug.
+
+![Notification Template](https://res.cloudinary.com/dxc6bnman/image/upload/f_auto,q_auto/v1685466980/guides/trigger_id_1_ur1azh.png)
+
+- The value of `Payload` is an array of the data that you want to be dynamically injected into the notification template content.
+- The value of `SubscriberId` is the id of the subscriber on Novu. Replace `7789` with your subscriber ID.
+
+Run the code to trigger a notification!
+
+```bash
+dotnet run Program.cs
+```
+
+Next, we’ll learn how to send notifications to different groups of subscribers easily via **Topics.**
+
+## Topics
+
+Novu provides a simple API that offers an easy interface for triggering notifications to multiple subscribers at once. This API is called **Topics** and allows users to manage their bulk notifications without having to implement complex loops.
+
+A topic is identified by a custom key and this key will be the identifier used when triggering notifications. You can assign a name to a topic for descriptive purposes. This name does not need to be unique and can be changed programmatically.
+
+::::info
+The topic key should be unique and can't be changed once chosen. Novu also safe guards for key uniqueness behind the scenes.
+::::
+
+A topic can have multiple subscribers who will receive a notification whenever a message is sent to the topic.
+
+## Create a Topic
+
+Copy and paste the following code into your app to create a topic:
+
+```c#
+// Create Topic
+var topicRequest = new TopicCreateDto
+{
+    Key = "frontend-users",
+    Name = "All frontende users",
+    
+};
+
+var topic = await client.Topic.CreateTopicAsync(topicRequest);
+
+
+```
+
+Before running the code, make sure you understand the following:
+
+- When creating a `Key`, ensure it is unique and accurately identifies the topic. Document naming conventions and communicate them to team members to avoid confusion and ensure a smooth workflow.
+- The value of `Name` should be a descriptive topic name.
+
+## Add subscribers to a Topic
+
+Copy and paste the following code into your app to add subscribers a topic:
+
+```c#
+// Add Subscriber to Topic
+var topicKey = "frontender-user";
+var subscriberList = new TopicSubscriberUpdateDto
+{
+    Keys = new List<string>
+    {
+        "7789",
+        "7790",
+    }
+};
+
+var result = await client.Topic.AddSubscriberAsync(topicKey, subscriberList);
+```
+
+## Sending a notification to a Topic
+
+Thanks to the topics feature, it is possible to trigger a notification to all subscribers assigned to a topic. This helps avoid listing all subscriber identifiers in the `to` field of the notification trigger.
+
+To trigger a notification to all subscribers of a topic, copy and paste the code below:
+
+Program.cs
+```c#
+// Send notifications to a topic (all frontend users)
+
+var onboardingMessage = new OnboardEventPayload
+{
+  Username = "jdoe",
```

Comment:
For this particular example, I don't think we need username. It won't be nice to send thesame username to all subscribers assigned to that topic